project/src/common/const.ts

```diff
@@ -11,6 +9,55 @@ const LogoSize = {
   },
 } as const;
 
+const LoaderParams = {
```

Comment:
```suggestion
const LoaderParam = {
```
енамы должны быть в единственном числе (исправить для всех других)